### PR TITLE
sync_diff_inspector: add unit test && print version information

### PR DESCRIPTION
**Pull request by WangXiangUSTC**

### CODE REVIEW

**Comment by WangXiangUSTC** [comment]
@GregoryIan @july2993  PTAL

File: pkg/db/common.go [hunk 1]
```diff
@@ -363,6 +363,38 @@ func GetCRC32Checksum(db *sql.DB, schemaName string, tbInfo *model.TableInfo, li
 	return "", nil
 }
 
+// GetTidbPosition gets tidb's position.
+func GetTidbPosition(db *sql.DB) (int64, error) {
```
Comment (WangXiangUSTC) [inline pkg/db/common.go:367]: this function is used in tidb-binlog

File: sync_diff_inspector/chunk_test.go [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+	expectChunks []*chunkRange
+}
+
+func (*testChunkSuite) TestSplitRange(c *C) {
+	testCases := []*chunkTestCase{
+		{
+			&chunkRange{
+				begin:        int64(1),
+				end:          int64(1000),
+				containBegin: true,
```
Comment (IANTHEREAL) [inline sync_diff_inspector/chunk_test.go:36]: Add [false,false] [false, true], [true, false] test cases

**Comment by IANTHEREAL** [comment]
found some bugs through unit tests, which is a good start, and keep the idea of improving your tests

**Comment by IANTHEREAL** [comment]
LGTM

File: sync_diff_inspector/chunk_test.go (hunk 2 above)
Comment (WangXiangUSTC) [reply sync_diff_inspector/chunk_test.go:36]: done

**Review by july2993 — APPROVED** [review]
LGTM

**Pull request merged**